rest-mvc/src/test/java/com/intuit/tank/rest/mvc/rest/controllers/AgentControllerTest.java

```diff
@@ -130,6 +132,15 @@ public void testAgentReady() {
         verify(agentService).agentReady(testAgentData);
     }
 
+    @Test
```

Comment:
@Zakaria-Kofiro No necessary a comment on this part of code, but are you looking to add unit tests the to service implementation side?

Reply:
Yes: https://jira.intuit.com/browse/SRE-26624

Reply:
Gotcha!